pom.xml

```diff
@@ -151,7 +151,7 @@
       <rabbitmq.http-client.version>3.12.1</rabbitmq.http-client.version>
 
       <!-- Vaadin versions - START -->
-      <vaadin.version>8.19.0</vaadin.version>
+      <vaadin.version>8.14.3</vaadin.version>
```

Comment:
Do you think we should add a note in the docs || code somewhere to indicate why we deliberately chose this version?